src/lib/ndpi_main.c

```diff
@@ -2759,6 +2763,9 @@ struct ndpi_detection_module_struct *ndpi_init_detection_module(ndpi_init_prefs
     return(NULL);
   }
 
+  ndpi_str->opportunistic_tls_smtp_enabled = 1;
+  ndpi_str->opportunistic_tls_ftp_enabled = 1;
```

Comment:
Is there any specific reason why TLS dissection for SMTP/FTP should be disabled?

Reply:
While this feature allow to extract some interesting metadata (fingerprints), in my experience it doesn't improve much the classification itself. Since it require more packets per flows, if you are not interested in such metadata, you can avoid the performance overhead